### ISSUE TITLE
[Bug]: Kubic repository missing dependency for CNI (or docs to fix missing)

### ISSUE THREAD

**Issue by computator** [containers/podman#17048]
### Issue Description

Using CNI networking with the latest podman (via the Kubic repository) in Ubuntu 22.04 does not work due to CNI not being installed. It seems to be missing a dependency on `containernetworking-plugins`, or documentation that this will need to be installed manually. Once installed everything works correctly.

### Steps to reproduce the issue

Steps to reproduce the issue
1. Remove old podman from 22.04 repos
2. Install podman from Kubic
3. Run `apt autoremove` (may not be necessary depending on circumstances)
4. Run a container or pod

### Describe the results you received

Podman fails to find the CNI plugins and errors out

### Describe the results you expected

Podman runs as normal

### podman info output

```shell
Podman 4.3.1 on x86_64 Ubuntu 22.04
```

### Podman in a container

No

### Privileged Or Rootless

Rootless

### Upstream Latest Release

No

**Comment by computator** [comment]
Forgot to note that on other computers I have it works fine with netavark. I kept CNI here though because I have containers I do not want to lose via running `podman system reset`

**Comment by Luap99** [comment]
We only need netavark/aardvark-dns on new systems since this is the new default with 4.0, if you need cni you have to install it manually. Installing both backends by default seems undesirable to me.
I am fine with documenting this but I have no idea where you would want this information?

**Comment by computator** [comment]
That seems totally reasonable to me to only have netavark as a dependency, I just wasn't sure if leaving out CNI was an oversight or intentional.

The logical place to document it to me is in the installation instructions, or if we have any, upgrade instructions, although I haven't seen any? Also probably wherever the docs discuss switching to CNI, although offhand I only recall seeing that in blog posts.

**Comment by rhatdan** [comment]
Interested in opening a PR to make this clearer?

**Comment by computator** [comment]
Yes I can probably do this but have yet to get to it

**Comment by rhatdan** [comment]
@computator still working on this?

**Comment by computator** [comment]
Yeah, I'll see if I can get to it this weekend or something, or if someone else wanted to do it that's fine too.
